Content.Client/Power/PowerSolarSystem.cs

```diff
@@ -0,0 +1,63 @@
+using Content.Shared.Atmos.Piping;
+using Content.Shared.Solar;
+using Content.Shared.VendingMachines;
+using JetBrains.Annotations;
+using Robust.Client.GameObjects;
+using Robust.Shared.Timing;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Content.Client.Power
+{
+    [UsedImplicitly]
+    public sealed class PowerSolarSystem : EntitySystem
+    {
+        // This is used for client-side prediction of the panel rotation.
+        // This makes the display feel a lot smoother.
+        private IGameTiming _gameTiming = IoCManager.Resolve<IGameTiming>();
```

Comment:
```suggestion
        [Dependency] private readonly IGameTiming _gameTiming = default!;
```

this will be injected when the entitysystem is created